example/local-alg-output.csv

```diff
@@ -4,7 +4,7 @@ input name,score,in-zone-traffic score,deviation score,slice score,max deviation
 2 zones with few endpoints,68.3333,50.0000,100.0000,66.6667,0.0000%,0.0000%,0.0000
 balanced minimal endpoints,86.6667,100.0000,100.0000,33.3333,0.0000%,0.0000%,0.0000
 unbalanced minimal endpoints,70.0000,66.6667,100.0000,33.3333,0.0000%,0.0000%,0.0000
-only 1 endpoint,46.6667,33.3333,33.3333,100.0000,66.6667%,66.6667%,1.3333
+only 1 endpoint,fail
```

Comment:
Is this a valid CSV? Can most parsers understand a variable number of columns/commas per row?